MapboxDirections/Attributes/MBSpeedLimit.swift

```diff
@@ -0,0 +1,118 @@
+import Foundation
+
+/**
+ A localized limit for measuring speed limits.
+ */
+@objc(MBSpeedLimit)
+public class SpeedLimit: NSObject, NSSecureCoding {
```

Comment:
Replace this class with a C struct to reduce overhead, and name it `MBSpeedLimit`. We can expect that the only fields it needs to contain are of type `double` and `MBSpeedUnit` (an enumeration), which can both be bridged from Objective-C to Swift. There would only be two reasons for making this a class: an instance method or a pointer-typed property, such as NSString.

Alternatively, drop support for iOS 8–9 and replace this class with the NSMeasurement class. `:daydream:`

Reply:
On second thought, we’re mainly putting these objects in an NSArray, and C structs need to be boxed in NSValue before going into an NSArray. So that sort of defeats the purpose of making them structs. For now, let’s make the class as similar to NSMeasurement as possible and revisit using NSMeasurement once we drop iOS 8–9 support.